spec/cypress/app_commands/scenarios/ect_cip.rb

```diff
@@ -5,6 +5,5 @@
 FactoryBot.create(:course_lesson, :with_lesson_part, course_module: course_module)
 
 # You have to create this user in your spec before running this scenario
-cip = User.find("53960d7f-1308-4de1-a56d-de03ea8e1d9c").core_induction_programme
-
-CoreInductionProgramme.find(cip.id).update!(course_year_one_id: year.id, course_year_two_id: year.id)
+cip = User.find("53960d7f-1308-4de1-a56d-de03ea8e1d9c")
+cip.core_induction_programme.update!(course_year_one_id: year.id, course_year_two_id: year.id)
```

Comment:
That's lying! 
`cip` is actually a user now - it used to be user's core induction programme, but you removed that 😂 
```
user = User.find("53960d7f-1308-4de1-a56d-de03ea8e1d9c")
user.core_induction_programme.update!(course_year_one_id: year.id, course_year_two_id: year.id)
```